_posts/2023-09-21-hybrid-search.md

```diff
@@ -0,0 +1,393 @@
+---
+layout: post
+title:  Improve search relevance with hybrid search, generally available in OpenSearch 2.10
+authors:
+  - gaievski
+  - navneev
+  - vamshin
+  - kolchfa
+date: 2023-09-21
+categories:
+  - technical-posts
+meta_keywords: 
```

Comment:
Please update with the following meta: 

**meta_keywords:**  Improve search relevance, hybrid search in OpenSearch 2.10, semantic and keyword search, 
**meta_description:**  Improve search relevance with OpenSearch 2.10 when you tune search relevance by using hybrid search to combine and normalize query relevance scores.